Calculator/calci.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="./calciStyle.css">
+</head>
+
+<body>
+  <div id="calculator">
+    <input id="display" readonly>
+    <div id="keys">
+      <button onclick="appendToDisplay('+')" class="operator-btn">+</button>
```

Comment:
Please use React and JSX

Reply:
Change the implementation from vanilla JS to react code